src/components/Tool/FixSendNewsletter/Contact.tsx

```diff
@@ -94,26 +80,31 @@ interface Props {
 }
 
 const Contact = ({
-  id,
-  name,
-  primaryPerson,
-  status,
-  primaryAddress,
+  contact,
   handleSingleConfirm,
   setContactFocus,
 }: Props): ReactElement => {
   const { t } = useTranslation();
   const [newsletter, setNewsletter] = useState(SendNewsletterEnum.None);
+  const [updatingSingle, setUpdatingSingle] = useState(false);
   const { classes } = useStyles();
+  const locale = useLocale();
 
+  const newsletterOptions = [
+    { value: SendNewsletterEnum.Physical, name: t('Physical') },
+    { value: SendNewsletterEnum.Email, name: t('Email') },
+    { value: SendNewsletterEnum.Both, name: t('Both') },
+    { value: SendNewsletterEnum.None, name: t('None') },
+  ];
+  const matches = useMediaQuery('(min-width:600px)');
   useEffect(() => {
     let newNewsletterValue = SendNewsletterEnum.None;
-    if (primaryAddress?.street) {
+    if (contact?.primaryAddress?.street) {
```

Comment:
I don't think you need the optional chaining on `contact?.primaryAddress` as contact has to be defined as per your type on this function.